man/avahi-daemon.8.xml.in

```diff
@@ -107,7 +107,7 @@
 		<optdesc><p>Tell an already running avahi-daemon to reread
 		<file>/etc/resolv.conf</file> (in case you enabled
         <opt>publish-resolv-conf-dns-servers</opt> in
-		<file>avahi-daemon.conf</file>) the files from
+		<file>avahi-daemon.conf</file>) and the files from
```

Comment:
Matches language here:
https://github.com/lathiat/avahi/blob/master/man/avahi-daemon.8.xml.in#L47-L48